packages/app-storage/src/Query.tsx

```diff
@@ -130,6 +135,36 @@ class Query extends React.PureComponent<Props, State> {
     );
   }
 
+  private renderInputs () {
+    const value = this.props.value;
+    const { key } = value;
+
+    // if function has inputs, render them
+    if ((value as StorageModuleQuery).params.length > 0) {
+      const type = isU8a(key) ? 'Data' : key.meta.type.asMap.key.toString();
+      const params = (value as StorageModuleQuery).params;
+      return (
+        <div>
+          Inputs: ({type})
+          {
+            params.map((obj) => {
+              // check if the type name references a tuple, e.g. "(Hash, AccountId)"
+              return (
+                <div className='query-input'>
```

Comment:
We would need a `key` attribute here to keep React happy and now spam to console, small enough.